src/Microsoft.OData.Core/ODataSimplifiedOptions.cs

```diff
@@ -88,6 +110,71 @@ public ODataSimplifiedOptions Clone()
             return copy;
         }
 
+        /// <summary>
+        /// Get whether to write OData control information without a prefix
+        /// True if control information can be read without prefix 'odata.', otherwise false.
+        /// The default value is false for OData 4.0 and true for OData 4.01.
+        /// The option is applied during deserialization.
+        /// </summary>
+        /// <returns>Whether to omit the OData prefix for the specified version.</returns>
+        public bool GetOmitODataPrefix()
+        {
+            return this.enableWritingODataAnnotationWithoutPrefix;
+        }
+
+        /// <summary>
+        /// Version-specific behavior for writing OData control information without a prefix
+        /// True if control information can be read without prefix 'odata.', otherwise false.
+        /// The default value is false for OData 4.0 and true for OData 4.01.
+        /// The option is applied during deserialization.
+        /// </summary>
+        /// <param name="version">The version of the version-specific behavior being requested.</param>
+        /// <returns>Whether to omit the OData prefix for the specified version.</returns>
+        public bool GetOmitODataPrefix(ODataVersion version)

```

Comment:
>ODataVersion version [](start = 39, length = 20)

why cannot use the "version" in the constructor? It's a little bit strange?

Reply:
ODataSimplifiedOptions is used by WebAPI OData to set options for the entire service. The challenge is that the same service may support both OData 4.0 and OData 4.01, and may want different settings of omitting the prefix for each.

---
In reply to: [279946255](https://github.com/OData/odata.net/pull/1454#discussion_r279946255) [](ancestors = 279946255)